incentivization.md

```diff
@@ -52,231 +47,186 @@ Early P2P file-sharing networks employed reputation-based approaches and sticky
 
 Potential issues:
-- An attacker can establish new server identities and continually run away with clients' money.
-	- countermeasures:
-		- a client only queries "trusted" servers (centralization);
-		- when querying a new server, a client first sends a small (i.e. cheap) request to not risk too much.
-- Think about how the ban mechanism can be abused. Can an attacker "frame" competitors' servers so that many clients ban them?
-
-### Results cross-checking
+- An attacker can establish new server identities and continue running away with clients' money. Countermeasures:
+	- a client only queries "trusted" servers (which however leads to centralization);
+	- when querying a new server, a client first sends a small (i.e. cheap) request as a test.
+- The ban mechanism can theoretically be abused. For instance, a competitor may attack the victim server and cause the clients who were awaiting the response to ban that server. Countermeasure: prevent DoS-attacks.
 
-> Never go to sea with two chronometers; take one or three.
+# Payment methods
 
-The client not only wants to receive _some_ response, it wants to receive all relevant messages and only them.
-We don't have consensus over history, so it's impossible to know for sure if a message is relevant.
-In non security-critical settings, a client may just accept the risk that some messages may be missing.
-For more certainty, the client may query 3 independent servers and compare the results.
-Only messages returned by 3/3 or 2/3 are considered relevant.
+The MVP protocol is agnostic to payment methods.
+A payment method should generally have the following properties:
+- wide distribution;
+- good liquidity;
+- low latency;
+- good privacy;
+- high security.
 
-Servers' reputation may then be adjusted, but it's not completely obvious how:
-- imagine a server whose response has a message that no other response has.
-	- should we punish it for inserting a fake message into history? or
-	- should we reward it for providing the data that other are (perhaps intentionally) hiding?
-- Same with a server that _misses_ some message that others have delivered: we don't know what the ground truth is anyway.
+Let's list all decentralized payment options:
+- ETH;
+- a token on Ethereum (ERC20);
+- a token on another EVM-based blockchain or a rollup;
+- a token on a non-EVM blockchain (such as BTC / Lightning).
 
-However, in the absence of a better mechanism, we can _define_ 2/3 as a validity criteria.
-Then, it follows that a server that does _not_ have the "2/3" message is either malicious (censors) or badly managed (went offline when this message was propagated).
-In any case, its reputation should be decreased.
+Note also that there may be different market models that may motivate the choice of the payment method.
+One model assumes that each client pays for its own requests.
+Another model includes (centralized?) entities (i.e., developers of Waku-based apps) that pay for their users in bulk.
```

Comment:
This may lead to a moral hazard situation where the free access may be overly excessive as it is free. There should be ways to define the scope of the free bandwidth.

Reply:
> free access may be overused as it is free

Are you referring to the model where a service provider makes the service free for its users by paying for them in bulk? In that case, it's up to that service to set fair use limits for its users. Also, there are only so many historical messages you can fetch.

Reply:
Clarified in the document, resolving the thread.